### PR TITLE
January 8th - Bombing occured in Baghdad

### PR DESCRIPTION
**Pull request by davidgtu**
Trying to learn how to contribute 👍

### CODE REVIEW

**Comment by alexbowers** [comment]
Thanks for the contribution. You have the process correct, however I would prefer a different source.

Currently, i'm only accepting BBC and Guardian sources, though this list will expand.

Please can you replace the source with one of those?

**Comment by davidgtu** [comment]
I've updated the sources and descriptions!

File: data/events.json [hunk 1]
```diff
@@ -61,5 +61,21 @@
...
+    },
+    {
+        "date": "08-01-2017",
+        "items": [
+            {
+                "title": "Car bomb explosion in Bagdad",
+                "description": [
+                    "The Islamic State (IS) has claimed responsibility behind the blast.",
+                    "The bomb was set off in a vegetable market.",
+                    "11 people have been killed and injuring dozens more."
```
Comment (alexbowers) [inline data/events.json]: 11 people have been killed and injuring dozens more. 

seems strangely phrased, 

"11 people have been killed and dozens more injured."

seems more natural.

**Comment by alexbowers** [comment]
Thanks, i'll merge after the final change requested! :)

File: data/events.json [hunk 2]
```diff
@@ -61,5 +61,21 @@
                 ]
             }
         ]
+    },
+    {
+        "date": "08-01-2017",
+        "items": [
+            {
```
Comment (alexbowers) [inline data/events.json:68]: Baghdad is misspelt here.

Comment (alexbowers) [reply data/events.json:68]: Also, standard format is to put `City, Country`

File: data/events.json (hunk 1 above)
Comment (alexbowers) [reply data/events.json]: Can we also reorder these sentences.

To me, the order of importance is:

- 11 people ...
- Vegetable Market
- IS

File: data/events.json [hunk 3]
```diff
@@ -61,5 +61,21 @@
...
+                "title": "Car bomb explosion in Baghdad",
+                "description": [
+                    "The Islamic State (IS) has claimed responsibility behind the blast.",
+                    "The bomb was set off in a vegetable market.",
+                    "11 people have been killed and dozens more injured."
```
Comment (alexbowers) [inline data/events.json:71]: Not sure if message about reordering came through:

To me, the order makes more sense being:

- 11 killed
- vegetable market
- IS

This keeps the most important information at the top.

Comment (davidgtu) [reply data/events.json:71]: Agreed. Making the changes now.

**Comment by alexbowers** [comment]
Thanks for all of the changes. The final one necessary is the `City, Country` ordering of the title

**Comment by alexbowers** [comment]
Thanks.

**Comment by alexbowers** [comment]
Oops. This had broken json in it. Guess i need to figure out a JSON linter to run before merges.

**Comment by davidgtu** [comment]
Oh shoot, I'll fix up the JSON and make another PR.

**Comment by alexbowers** [comment]
image

Already 3 visitors :)

**Comment by alexbowers** [comment]
JSON is now being properly validated on pull requests

**Pull request merged**